search-sentiments-map/public/index.html

```diff
@@ -20,17 +20,19 @@ <h2 id="title">Sentiment Map for Google Search</h2>
         <div id="map"></div>
       </div>
       <div class="col-sm-3">
-        <form id="search-box" class="input-group row" action="">
-          <input type="text" class="form-control col-sm-9" placeholder="Search a Topic" name="search">
-          <div class="input-group-append">
-            <button type="submit"><i class="fa fa-search"></i></button>
-          </div>
-        </form>
         <div>
-          <h3>Trending Search Topics</h3>
+          <h3>Trending Google Search Topics</h3>
```

Comment:
We are saying "Sentiment Map for Google Search" on the big title so I'll omit Google here